config-model/src/main/java/com/yahoo/vespa/model/content/ContentSearchCluster.java

```diff
@@ -336,6 +336,28 @@ public void handleRedundancy(Redundancy redundancy) {
                 .collect(Collectors.toList());
     }
 
+    public List<NewDocumentType> getDocumentTypesWithStreamingCluster() {
+        List<NewDocumentType> streamingDocTypes = new ArrayList<>();
+        for (NewDocumentType type : documentDefinitions.values()) {
+            if (findStreamingCluster(type.getFullName().getName()).isPresent()) {
+                streamingDocTypes.add(type);
+            }
+        }
+        return streamingDocTypes;
+    }
+
+    public List<NewDocumentType> getDocumentTypesWithIndexedCluster() {
+        List<NewDocumentType> indexedDocTypes = new ArrayList<>();
+        for (NewDocumentType type : documentDefinitions.values()) {
+            if (findStreamingCluster(type.getFullName().getName()).isEmpty()
```

Comment:
Not sure this first part is needed?

Reply:
It mimics the semantics of `getConfig`

Reply:
Yeah, if we need to separate streaming and store-only, we need it.